### ISSUE TITLE
Add syntax for defining custom SQL statements for datasets.

### ISSUE THREAD

**Issue by timothyrenner** [timothyrenner/svl#20]
This would enable a CSV to be transformed into multiple datasets.

```
DATASETS
bigfoot "bigfoot.csv"
bigfoot_locations SQL "SELECT * FROM bigfoot WHERE latitude IS NOT NULL"
```

**Comment by timothyrenner** [comment]
- [ ] Update grammar and parser with new syntax. Update test script to ensure parse is correct.
- [ ] Update all parser tests to pass with only files.
- [ ] Add parser tests for SQL statements.
- [ ] Implement SQL datasets in the data processor.
- [ ] Add SQL tests for data processor.
- [ ] End-to-end test against updated test script.

**Comment by timothyrenner** [comment]
Addressed in #27
